assignment1.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+	int choice = 0;
+	
+	cout << "Main menu:" << endl;
```

Comment:
Menus should always be inside of a while loop.  Use that instead, and you can remove the if statement at the bottom.